JS/Prototypes/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<p>Objects </p>
+
+
+
+<script>
```

Comment:
Always create a new file.
Move the below code to a new file and link it on this page.